jobs/payment-jobs/tasks/cfs_create_account_task.py

```diff
@@ -101,8 +101,12 @@ def _create_cfs_account(cls, pending_account: CfsAccountModel, pay_account: Paym
                 'bankAccountNumber': pending_account.bank_account_number,
                 'bankAccountName': pay_account.name
             }
-
-            if pending_account.cfs_account and pending_account.cfs_party and pending_account.cfs_site:
+            # switch from PAD to EFT
```

Comment:
Not sure what this comment means? Is the branch below only when you switch from PAD to EFT?

What happens if they create a new account that's EFT without being PAD first?

Reply:
removed